Gordon360/Models/ViewModels/HousingPreferredHallViewModel.cs

```diff
@@ -0,0 +1,21 @@
+using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class HousingPreferredHallViewModel
```

Comment:
Same here, what is the benefit of duplicating this database model as a ViewModel?